### PR TITLE
PUBDEV-8038: Expose compact way of representing contributions in Python

### PR DESCRIPTION
**Pull request by michalkurka**

### CODE REVIEW

File: h2o-py/h2o/model/model_base.py [hunk 1]
```diff
@@ -186,12 +186,17 @@ def predict_contributions(self, test_data):
         Note: Multinomial classification models are currently not supported.
 
         :param H2OFrame test_data: Data on which to calculate contributions.
+        :param Enum output_format: Specify how to output feature contributions in XGBoost - XGBoost by default outputs 
+            contributions for 1-hot encoded features, specifying a Compact output format will produce a per-feature
+            contribution. One of: ``"Original"``, ``"Compact"`` (default: ``"Compact"``).
```
Comment (g-eoj) [inline h2o-py/h2o/model/model_base.py:191]: Says default is compact but in code it's original.

**Review by g-eoj — APPROVED** [review]
Docstring issue, other than that LGTM. Thanks!

File: h2o-py/h2o/model/model_base.py (hunk 1 above)
Comment (michalkurka) [reply h2o-py/h2o/model/model_base.py]: Thank you for catching this. Fixed.

**Review by maurever — APPROVED** [review]
Looks good to me. Thanks, @michalkurka!

**Pull request merged**